src/Calendar.jsx

```diff
@@ -390,7 +388,7 @@ let Calendar = React.createClass({
     let ToolbarToRender = components.toolbar || Toolbar
 
     return (
-      <div {...elementProps}
```

Comment:
no something even simpler. the Calendar has a prop called: `elementProps` which is an object

 then you do: `<div {...this.props.elementProps>` on the calendar DOM node.